src/lint.js

```diff
@@ -60,6 +58,13 @@ function lintComments(comment: Comment) {
       checkCanonical(tag.type);
     }
   });
+
+  try {
+    nest(comment);
+  } catch (error) {
+    comment.errors.push({ message: error.message });
```

Comment:
@tmcw Not so good decision to use try ..catch. May be we need to rewrite nest module for lint usages without try..catch.